sappira-inc/quantumpack
# feat: Support Typescript

### 📃 Summary
Enable typescript via webpack configurations

### 🏷 Asana Task

### ↩️ Depends On

### 📸 Screenshots

### 🧪 QA
1) Run yarn example
2) make sure example app is running correctly.

### 📈 Risk Analysis
✅ Does this affect patients? no
✅ Does this affect clinical? no
✅ Does this mutate data? no
✅ Does it touch PHI? no
✅ Does it touch payments? no
